src/main/java/org/example/lambda/Lambda.java

```diff
@@ -0,0 +1,18 @@
+package org.example.lambda;
+
+import java.util.Arrays;
+
+public class Lambda {
+    public static void main(String[] args) {
+        String sentence = "This is a test sentence to check Lambda";
+        splitAndPrint(sentence);
+    }
+
+    public static void splitAndPrint(String sentence) {
+
```

Comment:
in main method you did not left empty lines after each line)) please stick to the one style, like this:
```suggestion
    public static void splitAndPrint(String sentence) {
        String[] words = sentence.split(" ");
        Arrays.asList(words).forEach(word -> System.out.println(word));
    }
```